app/assets/stylesheets/main.css.scss

```diff
@@ -52,3 +52,6 @@ $compatibility-high-color: #84C93F;
 @import 'styles/pages/home';
 @import 'styles/pages/styles';
 
+// Import modular CSS
+@import "modules/**/**/*";
+@import "modules/**/**/**/*";
```

Comment:
I think @simonv3 might be saying that the `@import`s are redundant since `@import "modules/**/*"` will already include ever file and folder under `modules/` recursively?